spec/batch.spec.js

```diff
@@ -350,6 +356,7 @@ describe('batch', () => {
               transaction: true,
             }),
           });
+          fail();
```

Comment:
could you try to refactor this as well, so we don't need the `fail`?

Reply:
This fail() should have been there in the first place and was a mistake when the test was originally added. I noticed the mistake while refactoring the test. If you look at the same exact test in batch.spec, it has the fail()

Reply:
Yes, I just thought you may want to refactor the test to not use `fail` but `expect.toThrow` since you are already modifying it. But either way it's an improvement.